lib/gengo-ruby/api_handler.rb

```diff
@@ -415,6 +415,23 @@ def deleteTranslationOrder(params = {})
       self.get_from_gengo('translate/order/:id'.gsub(':id', params.delete(:id).to_s), params)
     end
 
+    # Get all comments (the history) from a given order.
+    #
+    # Options:
+    # <tt>id</tt> - The ID of the order to get comments for.
+    def getTranslationOrderComments(params = {})
```

Comment:
Should accept `id` as a positional parameter since it does not make sense to get comments without order ID in this method.

The parameter should be named `order_id` rather than just `id`.